docs/demo/index.html

```diff
@@ -3,17 +3,22 @@
 <head>
    <meta charset=utf-8 />
    <title>silvermine-videojs-chromecast Demo</title>
-   <link href="https://unpkg.com/video.js@6.1.0/dist/video-js.css" rel="stylesheet">
-   <script src="https://unpkg.com/video.js@6.1.0/dist/video.js"></script>
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/video.js/7.11.6/video-js.min.css" />
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/video.js/7.11.6/video.js" ></script>
```

Comment:
Does this feature _require_ an upgrade to VideoJS? Making an upgrade to VideoJS is not something we take lightly, as it means all previous functionality of the plugin must be reviewed and re-tested - especially since this is a major upgrade.

If you're unsure, please test to see if the changes you're proposing are compatible with the current version of VideoJS this plugin is targeting.

Reply:
https://github.com/silvermine/videojs-chromecast/pull/112/commits/d7cd590e05153a015761f1d617754fa1f0ca35fb